src/WooCommerceShippingPackages.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/**
+ * @file
+ * Contains \Netzstrategen\ShopStandards\WooCommerceShippingPackages.
+ */
+
+namespace Netzstrategen\ShopStandards;
+
+/**
+ * Shipping related settings and actions.
+ */
+class WooCommerceShippingPackages {
+
+  /**
+   * Initialize the needed functions and hooks.
+   */
+  public static function init() {
+    // Show multiple shipping methods separately in order emails.
+    add_filter('woocommerce_get_order_item_totals', __CLASS__ . '::woocommerce_get_order_item_totals', 10, 2);
+  }
+
+  /**
+   * Calculates the shipping costs including tax.
+   *
+   * @param \WC_Order_Item_Shipping $shipping_method
+   *   A shipping item object derived from a WC_Order.
+   *
+   * @return string
+   *   The shipping costs including taxes, currency sign, and HTML line-breaks,
+   *   for output in the HTML order emails.
+   */
+  public static function getShippingCostsIncludingTaxes(\WC_Order_Item_Shipping $shipping_method): string {
+
+    if ($shipping_price) {
+      $currency = ' ' . get_woocommerce_currency_symbol();
+      $shipping_price = floatval($shipping_method->get_total());
+      $shipping_price_tax = floatval($shipping_method->get_total_tax());
+      $shipping_price_total = $shipping_price + $shipping_price_tax;
+      $shipping_price_total_with_symbol = ' - ' . $shipping_price_total . $currency;
```

Comment:
```suggestion
wc_price( $shipping_method->get_total() + $shipping_method->get_total_tax(), ['currency' => $order->get_currency()] );
```

Reply:
I am just following how they are doing it here: https://woocommerce.github.io/code-reference/files/woocommerce-includes-abstracts-abstract-wc-order.html#source-view.2012